Libraries/LibWeb/Bindings/MainThreadVM.cpp

```diff
@@ -725,7 +725,7 @@ void queue_mutation_observer_microtask(DOM::Document const& document)
     // 3. Queue a microtask to notify mutation observers.
     // NOTE: This uses the implied document concept. In the case of mutation observers, it is always done in a node
```

Comment:
I don't quite understand what this comment means, but I suppose it is probably no longer relevant?

Reply:
whoops, double up